src/shared/nested-dropdown-list/nested-dropdown-list-helper.ts

```diff
@@ -81,13 +85,13 @@ export namespace NestedDropdownListHelper {
 // =============================================================================
 const getInitialSubItem = <V1, V2, V3>(
     list: Map<string, CombinedFormattedOptionProps<V1, V2, V3>> | undefined
-): string[] => {
+): string[][] => {
```

Comment:
since `getInitialSubItem` is expected to always return only one item, the return type should remain as `string[]`. if the parent needs a 2d array, it can do its own wrapping into an array